brian2/parsing/rendering.py

```diff
@@ -287,15 +293,22 @@ class CPPNodeRenderer(NodeRenderer):
           # Bool ops
           'And': '&&',
           'Or': '||',
+          # C does not have a floor division operator (but see render_BinOp)
+          'FloorDiv': '/',
```

Comment:
We should also support FloorDiv on float values in C I think? At the moment it looks like ``1.0//2.0`` would get ``0.5`` in C whereas it should be ``0.0``.

Reply:
Agreed.